src/test/java/edu/cornell/kfs/module/purap/iwant/xml/TestIWantXmlObjects.java

```diff
@@ -0,0 +1,100 @@
+    
+    private static final String OUTPUT_FILE_PATH = "test/iwantdocxmltest/";
+    private static final String MAXIMO_OUTPUT_FILE_NAME = "maximoOutputFile.xml";
+    private static final String IWANT_OUTPUT_FILE_NAME = "iwantOutputFile.xml";
+    
+    private File outputFileDirectory;
+    
+    private CUMarshalServiceImpl marshalservice;
+
+    @BeforeEach
+    void setUp() throws Exception {
+        marshalservice = new CUMarshalServiceImpl();
+        outputFileDirectory = new File(OUTPUT_FILE_PATH);
+        FileUtils.forceMkdir(outputFileDirectory);
+    }
+
+    @AfterEach
+    void tearDown() throws Exception {
+        marshalservice = null;
+        FileUtils.forceDelete(outputFileDirectory.getAbsoluteFile());
+    }
+
+    @Test
+    public void testIWantMaximoFileAgainstMarshalledFile() throws JAXBException, IOException {
+        File maximoFile = new File(INPUT_FILE_PATH + MAXIMO_INPUT_FILE_NAME);
+
+        IWantDocumentWrapperXml iWantDocFile = marshalservice.unmarshalFile(maximoFile, IWantDocumentWrapperXml.class);
+        LOG.info("testMaximoExample, iWantDocFile: " + iWantDocFile.toString());
+        String outputFileName = OUTPUT_FILE_PATH + MAXIMO_OUTPUT_FILE_NAME;
+        marshalservice.marshalObjectToXMLFragment(iWantDocFile, outputFileName);
+        
+        File outputMaximoFile = new File(outputFileName);
+        
+        compareXmlFiles(maximoFile, outputMaximoFile);
+    }
+    
+    @Test
+    public void testIWantExampleFileAgainstMarshalledFile() throws JAXBException, IOException {
+        File iwantExampleFile = new File(INPUT_FILE_PATH + IWANT_INPUT_FILE_NAME);
+        
+        IWantDocumentWrapperXml docFile = IWantDocumentWrapperFixture.FULL_EXAMPLE.toIWantDocumentWrapperXml();
+        LOG.info("testIwantDocFromObject. docFile : " + docFile.toString());
+        
+        String outputFileName = OUTPUT_FILE_PATH + IWANT_OUTPUT_FILE_NAME;
+        marshalservice.marshalObjectToXMLFragment(docFile, outputFileName);
+        
+        File outputIwantFullExample = new File(outputFileName);
+        
+        compareXmlFiles(iwantExampleFile, outputIwantFullExample);
+        
+    }
+
+    private void compareXmlFiles(File iwantExampleFile, File outputIwantFullExample) {
+        CuXMLUnitTestUtils.compareXMLWithEvaluators(iwantExampleFile, outputIwantFullExample, 
+                DifferenceEvaluators.Default, new KualiDecimalXmlDifferenceEvaluator());
+    }
+    
+    @Test
+    public void testIWantExamplePojoForUnmarshalledFile() throws JAXBException {
+        File iwantExampleFile = new File(INPUT_FILE_PATH + IWANT_INPUT_FILE_NAME);
+        IWantDocumentWrapperXml actualDocumentWrapper = marshalservice.unmarshalFile(iwantExampleFile, IWantDocumentWrapperXml.class);
+        
+        IWantDocumentWrapperXml expectedDocumentWrapper = IWantDocumentWrapperFixture.FULL_EXAMPLE.toIWantDocumentWrapperXml();
+        
+        assertEquals(actualDocumentWrapper, expectedDocumentWrapper);
```

Comment:
The `assertEquals()` method treats the first argument as the "expected" value and the second argument as the "actual" value. Please adjust the order of these arguments accordingly.

Reply:
ok